netket/nn/symmetric_linear.py

```diff
@@ -211,35 +572,213 @@ def full_bias(self, bias):
         return jnp.repeat(bias, self.n_symm)
 
     @compact
-    def __call__(self, inputs: Array) -> Array:
+    def __call__(self, x: Array) -> Array:
         """Applies the equivariant transform to the inputs along the last dimension.
         Args:
-          inputs: The nd-array to be transformed.
+          x: The nd-array to be transformed.
         Returns:
           The transformed input.
         """
-        dtype = jnp.promote_types(inputs.dtype, self.dtype)
-        inputs = jnp.asarray(inputs, dtype)
+        dtype = jnp.promote_types(x.dtype, self.dtype)
+        x = jnp.asarray(x, dtype)
+
+        x = x.reshape(-1, x.shape[1] * x.shape[2])
 
         kernel = self.param(
             "kernel",
             self.kernel_init,
-            (inputs.shape[-1] // self.in_features, self.in_features, self.out_features),
+            (self.out_features, self.in_features, self.n_symm),
             self.dtype,
         )
+
+        kernel = jnp.asarray(kernel, dtype)
+
+        if self.mask:
+            kernel = kernel * jnp.expand_dims(self.mask, (0, 1))
+
         kernel = self.full_kernel(kernel)
         kernel = jnp.asarray(kernel, dtype)
 
-        y = lax.dot_general(
-            inputs,
+        x = lax.dot_general(
+            x,
             kernel,
-            (((inputs.ndim - 1,), (0,)), ((), ())),
+            (((x.ndim - 1,), (0,)), ((), ())),
             precision=self.precision,
         )
 
         if self.use_bias:
             bias = self.param("bias", self.bias_init, (self.out_features,), self.dtype)
             bias = jnp.asarray(self.full_bias(bias), dtype)
-            y += bias
+            x += bias
 
-        return y
+        return x
+
+
+def DenseSymm(symmetries, mode="auto", shape=None, point_group=None, **kwargs):
+    """
+    Implements a projection onto a symmetry group. The output will be
+    equivariant with respect to the symmetry operations in the group and can
+    be averaged to produce an invariant model.
+
+    Args:
+        symmetries: A specification of the symmetry group. Can be given by a
```

Comment:
The order of the arguments in the docstring here does not match the actual ordering of the function

Reply:
moreover, notice that ```shape``` is not defined in the docstring

Reply:
This should all be fixed now!